src/style/_common.scss

```diff
@@ -1,4 +1,25 @@
 .body{
-    padding: 0;
+    padding-left: 240px;
+}
+
+.header{
+    width: 100%;
+    height: 200px;
+    background-color: black;
     margin: 0;
-}
\ No newline at end of file
+}
+
+.page-header{
+    background-image: url(/src/img/backgroundheader.jpg);
+
+}
+
+.flex-grow-1{
+    margin-left: 240px;
+}
+
+.flex-grow-2{
+    margin-left: 35%;
```

Comment:
Flex grow class should be responsive for flex-grow property only. Please user another class